component/core/src/main/java/org/exoplatform/social/core/image/ImageUtils.java

```diff
@@ -128,27 +132,75 @@ public static AvatarAttachment createResizedAvatarAttachment(InputStream imageSt
       }
 
       // Create temp file to store resized image to put to avatar attachment
-      File tmp = File.createTempFile("RESIZED", null);
+      ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
```

Comment:
may be it's better to not change here, sorry I wasn't clear may be about this change which finally should be applied on Default Avatars only to not have to bring huge avatar files in memory, in contrary to default avatars which is less heavy.

Reply:
fixed